crosshair/libimpl/builtinslib.py

```diff
@@ -1967,6 +1990,24 @@ def split(self, sep: Optional[str] = None, maxsplit: int = -1):
         ret.extend(self[first_occurance + 1 :].split(sep=sep, maxsplit=new_maxsplit))
         return ret
 
+    def rsplit(self, sep: Optional[str] = None, maxsplit: int = -2):
```

Comment:
Is the default of maxsplit supposed to be -1?

Reply:
Oops